src/components/companies-list/companies-list.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 
-import CompaniesListItem from '../company-list-item/companies-list-item.js';
+import {CompaniesListItem} from '../company-list-item/company-list-item.js';
 
-export const CompaniesList = ({ posts, onDelete }) => (
+export const CompaniesList = ({ posts, onDelete, onEdit }) => (
     <div>
         {posts.map(({ id, ...itemProps }) =>
             <CompaniesListItem
                 {...itemProps}
-                key={id}                
-                onDelete={() => onDelete(id)}
+                key={id}      
+                onEdit={onEdit}          
```

Comment:
Make onEdit and onDelete props in the same styles

Reply:
+

Reply:
yes

Reply:
done